pkgs/desktops/lomiri/development/libusermetrics/default.nix

```diff
@@ -98,15 +98,9 @@ stdenv.mkDerivation (finalAttrs: {
 
   doCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
 
-  checkPhase = ''
-    runHook preCheck
-
-    export QT_PLUGIN_PATH=${lib.getBin qtbase}/lib/qt-${qtbase.version}/plugins/
-    export QML2_IMPORT_PATH=${lib.getBin qtdeclarative}/lib/qt-${qtbase.version}/qml/
-    dbus-run-session --config-file=${dbus}/share/dbus-1/session.conf -- \
-      make test "''${enableParallelChecking:+-j $NIX_BUILD_CORES}"
```

Comment:
you are no longer executing the tests. why?

Reply:
Tests still run on OfBorg. What we are no longer doing is manually executing them under `dbus-run-session` instead of relying on the regular `checkPhase`, because it seems like that's not actually required for this package.